test/Datadog.Trace.ClrProfiler.IntegrationTests/GraphQLTests.cs

```diff
@@ -168,55 +201,69 @@ private void SubmitRequests(int aspNetCorePort)
         {
             foreach (RequestInfo requestInfo in _requests)
             {
-                try
-                {
-                    var request = WebRequest.Create($"http://localhost:{aspNetCorePort}{requestInfo.Url}");
-                    request.Method = requestInfo.HttpMethod;
+                SubmitRequest(aspNetCorePort, requestInfo);
+            }
+        }
 
-                    if (requestInfo.RequestBody != null)
-                    {
-                        byte[] requestBytes = System.Text.Encoding.UTF8.GetBytes(requestInfo.RequestBody);
+        private HttpStatusCode SubmitRequest(int aspNetCorePort, RequestInfo requestInfo, bool printResponseText = true)
+        {
+            try
+            {
+                var request = WebRequest.Create($"http://localhost:{aspNetCorePort}{requestInfo.Url}");
```

Comment:
Unrelated to this PR, but we should have used `HttpClient` instead of the [deprecated](https://docs.microsoft.com/en-us/dotnet/api/system.net.webrequest?view=netframework-4.8#remarks) `WebRequest`.